server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -167,11 +206,21 @@ public Response loadTable(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "alter-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "alter-table", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "( (METALAKE::USE_CATALOG || CATALOG::USE_CATALOG) && "
+              + "(METALAKE::USE_SCHEMA || CATALOG::USE_SCHEMA ||SCHEMA::USE_SCHEMA) &&"
+              + " (METALAKE::MODIFY_TABLE || CATALOG::MODIFY_TABLE || "
+              + "SCHEMA::MODIFY_TABLE || TABLE::MODIFY_TABLE || TABLE::OWNER)) ||"
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files